sympy/polys/tests/test_numberfields.py

```diff
@@ -637,9 +637,9 @@ def test_AlgebraicNumber():
 
     a = AlgebraicNumber(sqrt(2))
     b = to_number_field(sqrt(2))
-    assert a.args == b.args == (sqrt(2), Tuple())
+    assert a.args == b.args == (sqrt(2), Tuple(1, 0))
```

Comment:
I think that ``None`` should be preferred as default, at least for now. It is then possible to choose any coefficients at will whenever required. I also added a test with ``(0, 0)``. There was one with an empty list already. 

I do not understand the purpose of the sign change. It makes all negative real roots positive but why... I have not found any explanation. That may have to be changed but perhaps not in this PR. Anyway, it is always possible to avoid the change by giving explicit ``coeffs``.